documentation/templates/plan_functions.md.erb

```diff
@@ -41,7 +40,7 @@ This function<%= func['signatures'].count == 1 ? '' : ' signature' %> accepts th
 | --- | --- | --- |
 <% sig['options'].each do |name, data| -%>
 | `<%= name %>` | `<%= data['type'] %>` | <%= data['desc'] %> |
-<% end -%>
+<% end %>
```

Comment:
Er, doesn't this need the `-%>`?

Reply:
Not really. The page will still render fine on the docs site, but this fixes an issue where the markdown wasn't displaying correctly because there wasn't a blank line between the end of the table and the next line of text.